fcrepo-search-impl/src/main/resources/sql/default-search-index.sql

```diff
@@ -1,8 +1,23 @@
 -- simple search
 CREATE TABLE IF NOT EXISTS simple_search (
-    fedora_id  varchar(503) NOT NULL PRIMARY KEY,
+    id bigint PRIMARY KEY AUTO_INCREMENT,
+    fedora_id  varchar(503) NOT NULL,
     created timestamp NOT NULL,
     modified timestamp NOT NULL,
     content_size bigint DEFAULT NULL,
-    mime_type varchar(255) DEFAULT NULL
-);
\ No newline at end of file
+    mime_type varchar(255) DEFAULT NULL,
+    UNIQUE KEY fedora_id (fedora_id)
+);
+
+CREATE TABLE IF NOT EXISTS search_resource_rdf_type (
+    resource_id bigint NOT NULL,
```

Comment:
Why do these DDLs not have any foreign key relationships?

Reply:
I ran into an issue with truncation failing when I had foreign keys on there.

Reply:
You need to use `CASCADE` when you truncate a table with foreign keys.

Reply:
I will try that - thanks for the tip.